### ISSUE TITLE
audio

### ISSUE THREAD

**Issue by Karol196** [LunaPixelStudios/Better-MC#1459]
### Describe the Bug

I don;t hear anything when i am playing Better Minecraft 1.16.5 but i can hear other games videos and on other versions everything is working 

### Reproduction Steps

1.I don't know its just not working
2.
3.
...

### Screenshots and Videos

i just don;t hear anything ;/

### Operating System

Windows 11

### Minecraft Version

1.16

### Modloader

Forge

### Modpack Version

1.16.5

### Other Mods

no i am using only urs

**Comment by RaneNay** [comment]
I also have this issue, same everything. 
Recently, over the past 4 days I have had no audio in BMC1 1.16.5 v60.5, It had worked before and I had gotten on one day with no audio. I had temporarily fixed the problem by transferring a new OpenAL.dll but that only worked for a while until I came back to my computer the following day after shutting it down and turning it back on. I have restarted my pc several times, uninstalled and reinstalled all game files and even curseforge, I have used f3+t and f3+s, my mixers are not muted, and both directional and spatial audio is off, I have adjusted my minimap sizes, I have updated all of my drivers.

I have tested two different audio devices and still no audio. 

Other versions of BMC, including vanilla do have audio.

I have also noticed upon launching out of curseforge to an MC launcher, the MC launcher does not have audio that automatically play, by does play videos with audio.
